raiden/constants.py

```diff
@@ -21,6 +21,9 @@
 RED_EYES_PER_TOKEN_NETWORK_LIMIT = int(250 * 10 ** 18)
 
 GENESIS_BLOCK_NUMBER = 0
+STATE_PRUNING_AFTER_BLOCKS = 128
```

Comment:
This should be client specific, no?

Reply:
Hmm this is hard coded in geth. But in parity it's configurable .. so the clear solution here is to make this also configurable but I would not make this part of this PR.